try.html

```diff
@@ -2089,12 +2089,19 @@ <h3 id="miscellaneous">Miscellaneous</h3>
           <td><code>https://img.shields.io/vscode-marketplace/r/ritwickdey.LiveServer.svg</code></td>
         </tr>
         <tr>
-          <th data-keywords="eclipse marketplace">Eclipse Marketplace:</th>
+          <th data-keywords="eclipse marketplace favorites">Eclipse Marketplace favorites:</th>
```

Comment:
After #1163 `try.html` is generated, so please make these changes in [lib/all-badge-examples.js](https://github.com/badges/shields/blob/master/lib/all-badge-examples.js) instead, and then run `npm run build`.

`try.html` will no longer be tracked after #1194.

Reply:
Done, will keep that in mind in the future!